packages/components/src/components/DistributionsChart/index.tsx

```diff
@@ -177,7 +131,7 @@ const InnerMultiDistributionChart: FC<{
           context.textBaseline = "middle";
           context.fillStyle = "black";
           context.font = "12px sans-serif";
-          context.fillText(shapes[i].name, 16, radius);
+          context.fillText(shapes[i].name ?? "TODO", 16, radius);
```

Comment:
At least a longer message, if you're doing this in this PR? Like, "No Name", with a comment saying "TODO"

Reply:
Oops. I forgot to fix this.
It won't ever be triggered, though, because `isMulti` is only possible when the dist is built with `Plot.dists`, which requires names.
Anyway, fixed now.